### PR TITLE
Start adding sitemap documentation

### PR DESCRIPTION
**Pull request by SariSaar**

### CODE REVIEW

File: src/docs/concepts-development/sitemap/index.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+However, on a marketplace, it is likely that you will have listing URLs
+that include dynamic data from each listing – and you still want search
+engines to index those pages.
+
+<info>
+
+The Sharetribe Web Template has tooling to create a dynamic sitemap by
+default. Read more about the template sitemap structure and logic:
+**[Sitemap in Sharetribe Web Template](/sitemap-in-template)**
```
Comment (Gnito) [inline src/docs/concepts-development/sitemap/index.md:25]:
```suggestion
**[Sitemap in Sharetribe Web Template](/ftw/sitemap-in-template/)**
```
Linkki ei toiminut.

File: src/docs/ftw-analytics/template-sitemap/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+When running the template locally with _yarn run dev_, you can find
+these routes on the dev server port, i.e. localhost:3500.
+
+You can read more about the template sitemap and other resources in the
+[Resources README](https://github.com/sharetribe/web-template/blob/main/server/resources/README.md).
+
+### robots.txt
+
+The [your-domain]**/robots.txt** file details the authenticated routes
+that crawlers should avoid, and exposes the URL to the main sitemap. If
```
Comment (Gnito) [inline src/docs/ftw-analytics/template-sitemap/index.md:31]:
```suggestion
The [your-domain]**/robots.txt** file details the authenticated routes
as disallowed for crawlers, and exposes the URL to the main sitemap. If
```

File: src/docs/ftw-analytics/template-sitemap/index.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+you custom develop routes that require authentication, it is a good idea
+to specify them here.
+
+### sitemap-index.xml
+
+The [your-domain]**/sitemap-index.xml** file further exposes three
+separate sub-sitemaps. This is useful because it allows you more
+detailed tracking in analytics tools, and it reduces the size of each
+individual sitemap.
+
```
Comment (Gnito) [inline src/docs/ftw-analytics/template-sitemap/index.md:41]:
```suggestion

This is the sitemap, you should give to [Google Search Console](https://www.sharetribe.com/help/en/articles/8413179-how-to-configure-google-search-console).

```

The link now leads to an article about how to configure the Google Search Console, but there might be a specific article written later on about how to feed the sitemap to Google Search Console.

File: src/docs/ftw-analytics/template-sitemap/index.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+[server/resources/sitemap.js](https://github.com/sharetribe/web-template/blob/main/server/resources/sitemap.js#L284)
+
+## Sitemap caching
+
+The template caches sitemap data
+[for one day by default](https://github.com/sharetribe/web-template/blob/main/server/resources/sitemap.js#L55).
+Especially on bigger sites with multiple listings, listing sitemap data
+can take up to a few seconds to fetch from the SDK. Caching the results
+improves the sitemap performance.
+
```
Comment (Gnito) [inline src/docs/ftw-analytics/template-sitemap/index.md:104]: Also, API will cache listings & pages endpoint. Currently, it's 1 day too, which means that the combined caching effect might be a maximum of 2 days on top of the sharetribe/web-template.

File: src/docs/concepts-development/sitemap/index.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+slug: sitemap-in-sharetribe
+updated: 2023-10-12
+category: concepts-development
+ingress:
+  This guide describes how the Sharetribe tooling supports creating a
+  sitemap for your marketplace.
+published: true
+---
+
+Having a sitemap on your marketplace helps search engines to process
```
Comment (shareoc) [inline src/docs/concepts-development/sitemap/index.md:12]:
```suggestion
Having a sitemap on your marketplace helps search engines crawl and index
```

File: src/docs/concepts-development/sitemap/index.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+title: Sitemap support in Sharetribe
+slug: sitemap-in-sharetribe
+updated: 2023-10-12
+category: concepts-development
+ingress:
+  This guide describes how the Sharetribe tooling supports creating a
+  sitemap for your marketplace.
+published: true
+---
+
```
Comment (Gnito) [inline src/docs/concepts-development/sitemap/index.md:11]: Useful links for people who read this article:

- https://developers.google.com/search/docs/crawling-indexing/sitemaps/build-sitemap#xml
- https://developers.google.com/search/docs/crawling-indexing/sitemaps/large-sitemaps
- https://developers.google.com/search/docs/crawling-indexing/robots/intro

File: src/docs/concepts-development/sitemap/index.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+</info>
+
+If you are creating your own custom application, you can consider
+whether you want to use the same practices that the web template uses to
+craft the sitemap.
+
+## Separate sitemaps for static and dynamic content
+
+The web template exposes three different sub-sitemaps from
```
Comment (shareoc) [inline src/docs/concepts-development/sitemap/index.md:35]: Comment:
How this is worded makes it seem that this information is web-template specific. Is the idea here to use the approach the web-template uses as an example for non-template developers? What do you think about using a more general approach, e.g.:

Static content sitemaps help search engines quickly index unchanging pages like your "About" page, while dynamic content sitemaps focus on frequently updated or user-generated content, such as listings in your marketplace. For example, the Sharetribe Web Template generates three different sub-sitemaps linked from _sitemap-index.xml_, all with slightly different data:

File: src/docs/concepts-development/sitemap/index.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+  This guide describes how the Sharetribe tooling supports creating a
+  sitemap for your marketplace.
+published: true
+---
+
+Having a sitemap on your marketplace helps search engines to process
+your website correctly. For a traditional website, sitemaps are
+relatively easy to make, because all of your routes are built in to the
+application.
+
```
Comment (shareoc) [inline src/docs/concepts-development/sitemap/index.md:16]: Comment:
Maybe we could mention that we provide API endpoints (and link to them) for generating dynamic sitemaps already in the introduction section right now they're only mentioned under "Separate sitemaps for static and dynamic content"

File: src/docs/concepts-development/sitemap/index.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+The web template exposes three different sub-sitemaps from
+_sitemap-index.xml_, all with slightly different data:
+
+- _sitemap-default.xml_ exposes public built-in pages in the template.
+  Non-public routes that require authentication are disallowed in the
+  _robots.txt_ file.
+- _sitemap-recent-listings.xml_ exposes routes to the most recently
+  created listings. The listing data is fetched from a specific
+  [_sitemapData_ endpoint](TODO API REF LINK). The endpoint returns the
+  10 000 most recent listings.
```
Comment (shareoc) [inline src/docs/concepts-development/sitemap/index.md:44]: I couldn't find the answer via the API ref, but maybe we could have an answer for the question "what if I have more than 10 000 listings in my marketplace" here

File: src/docs/ftw-analytics/template-sitemap/index.md (hunk 3 above)
Comment (SariSaar) [reply src/docs/ftw-analytics/template-sitemap/index.md:41]: There is a section with an explanation and a link related to Google search console at the end of this article

File: src/docs/concepts-development/sitemap/index.md (hunk 9 above)
Comment (SariSaar) [reply src/docs/concepts-development/sitemap/index.md:44]: Great question, asked [in Slack](https://sharetribe.slack.com/archives/C03MHA74QQ4/p1697118379278729)

Comment (SariSaar) [reply src/docs/concepts-development/sitemap/index.md:52]: No pagination in the sitemap endpoints, so folks who want to include listings beyond 10k will need to fetch the older listings through Integration API. I'll include a note on this in the article!

**Pull request merged**